gio/src/auto/functions.rs

```diff
@@ -8,6 +8,7 @@ use crate::Cancellable;
 use crate::DBusConnection;
 use crate::File;
 use crate::IOErrorEnum;
+use crate::IOModule;
```

Comment:
```
 error[E0432]: unresolved import `crate::IOModule`
Error:   --> gio\src\auto\functions.rs:11:5
   |
11 | use crate::IOModule;
   |     ^^^^^^^^^^^^^^^ no `IOModule` in the root
```

Reply:
heh a gir issue, if a parameter is behind a feature flag; the function & its imports are not behind that feature flag

Reply:
Please open an issue if not already done. ;)